src/components/LoginDialogueBox.tsx

```diff
@@ -22,8 +64,40 @@ const LoginDialogueBox = (props: DialogueProps) => {
     setPassword(e.target.value);
   };
 
+  const handleSubmit = async (e: SyntheticEvent) => {
```

Comment:
ideally, as a user, I would just have a username, and that would be associated to a nonprofit or a volunteer, so you in your API could just know automatically